static/app/views/performance/landing/index.tsx

```diff
@@ -129,19 +133,30 @@ export function PerformanceLanding(props: Props) {
                 currentFilter={spanFilter}
                 onChangeFilter={setSpanFilter}
               />
-              <SearchBar
-                searchSource="performance_landing"
-                organization={organization}
-                projectIds={eventView.project}
-                query={filterString}
-                fields={generateAggregateFields(
-                  organization,
-                  [...eventView.fields, {field: 'tps()'}],
-                  ['epm()', 'eps()']
-                )}
-                onSearch={handleSearch}
-                maxQueryLength={MAX_QUERY_LENGTH}
-              />
+              {isMetricsData && defined(eventView.project[0]) ? (
```

Comment:
Metrics has to have an `id`? So `-1` for all projects is fine, but not 2 separate selected projects?

Reply:
I will update this code as soon as this PR is merged https://github.com/getsentry/sentry/pull/30088. With these changes the metrics API will support multiple projects